### PR TITLE
Try CI fix

### PR DESCRIPTION
**Pull request by Warhedgehog**
This fixes temporarly the CI problems. 

Because of the formatting it seems like it changed everything but I've highlighted the parts I've actually changed

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -107,58 +106,61 @@ jobs:
...
+        run: make check-fmt ; if [ "$(gofmt -s -l ./ | wc -l)" -gt 0 ]; then exit 1; fi
+
+      - name: Run check target
+        run: |
+          make check
+      - uses: dominikh/staticcheck-action@v1.1.0
+        with:
+          version: "2021.1.1"
+          install-go: false
+          cache-key: ${{ matrix.go }}
```
Comment (Warhedgehog) [inline .github/workflows/ci.yaml:146]: This was changed

File: be1-go/Makefile [hunk 2]
```diff
@@ -4,9 +4,7 @@ build: protocol
 	go build -o pop ./cli
 
 lint:
-	@go get -v honnef.co/go/tools/cmd/staticcheck
 	@go mod tidy
-	staticcheck ./...
```
Comment (Warhedgehog) [inline be1-go/Makefile:9]: This was changed

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -107,58 +106,61 @@ jobs:
...
-      uses: SonarSource/sonarcloud-github-action@master
-      with:
-        projectBaseDir: be1-go
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
-        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN_BE1 }}
+      - name: Setup Go 1.17.8 #Temporary fix to get the CI running again for now (5.4.2022)
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.8
```
Comment (Warhedgehog) [inline .github/workflows/ci.yaml:112]: This was changed

File: .github/workflows/ci.yaml [hunk 4]
```diff
@@ -107,58 +106,61 @@ jobs:
...
+        run: |
+          make check
+
+      - name: Run go vet
+        run: |
+          make vet
+
+      - uses: dominikh/staticcheck-action@v1.1.0
+        with:
+          version: "2021.1.1"
```
Comment (Warhedgehog) [inline .github/workflows/ci.yaml:149]: This was added

**Comment by lhmerino** [comment]
Looks good to me and I'll approve the pull request; if you need someone with knowledge about the CI, please wait for someone else to approve the pull request before merging.

**Review by lhmerino — APPROVED** [review]

**Review by Robingoumaz — APPROVED** [review]

**Review by nkcr — APPROVED** [review]

**Pull request merged**